src/makeRobotAccountant.js

```diff
@@ -6,7 +6,20 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callCount = 0;
+
+  const getSum = (firstNumber) => {
+    callCount++;
+
+    return (secondNumber) => {
+      const isEvenCall = callCount % 2 === 0;
+      const isInvalidCall = isEvenCall && callCount > 3;
+
+      return isInvalidCall ? 'Bzzz... Error!' : firstNumber + secondNumber;
+    };
+  };
```

Comment:
Let's increment the counter only when the logic is executed to avoid unexpected results

```suggestion
  const getSum = (firstNumber) => {
    return (secondNumber) => {
      callCount++;
  
      const isEvenCall = callCount % 2 === 0;
      const isInvalidCall = isEvenCall && callCount > 3;

      return isInvalidCall ? 'Bzzz... Error!' : firstNumber + secondNumber;
    };
  };
```